### ISSUE TITLE
Not receiving Hound comments

### ISSUE THREAD

**Issue by nneal** [houndci/eslint#23]
Unfortunately, we have never been able to get hound comments from eslint (everything else works great). Any idea? Note we have a lot of JSX files, so maybe that format could be part of the problem?

eslint local version `v1.10.3`

```
ruby:
  config_file: .ruby-style.yml
javascript:
  enabled: false
eslint:
  enabled: true
  config_file: .eslintrc
haml:
  enabled: true
  config_file: .haml-style.yml
```

eslintrc

```
{
  "extends": "airbnb",
  "globals": {
    "_": true
  },
  "env": {
    "jquery": true
  }
}
```

**Comment by timaschew** [comment]
maybe related: #27

**Comment by gylaz** [comment]
I believe this is resolved by #25.
